tmux.conf

```diff
@@ -55,13 +55,22 @@ bind-key T previous-window
 bind-key [ copy-mode
 bind-key ] paste-buffer
 
-# Setup 'v' to begin selection as in Vim
-bind-key -t vi-copy v begin-selection
-bind-key -t vi-copy y copy-pipe "reattach-to-user-namespace pbcopy"
+run-shell "tmux setenv -g TMUX_VERSION $(tmux -V | cut -c 6-)"
 
+# Setup 'v' to begin selection as in Vim
 # Update default binding of `Enter` to also use copy-pipe
-unbind -t vi-copy Enter
-bind-key -t vi-copy Enter copy-pipe "reattach-to-user-namespace pbcopy"
+#
+# New keybindings for vi-mode when version >= 2.4
+# https://github.com/tmux/tmux/issues/754
+if-shell -b '[ "$(echo "$TMUX_VERSION >= 2.4" | bc)" = 1 ]' \
```

Comment:
Doesn't this if statement need another leg to include the config in case tmux version is < 2.4? I'm confused.